source/kraken/tests/realtime_test.cpp

```diff
@@ -2350,7 +2353,8 @@ BOOST_AUTO_TEST_CASE(add_modify_and_delete_new_stop_time_in_the_trip) {
                     RTStopTime("stop_point:B", "20171101T0830"_pts),
                     RTStopTime("stop_point:B_bis", "20171101T0845"_pts).skipped(),
                     RTStopTime("stop_point:C", "20171101T0900"_pts),
-            });
+            },
+            transit_realtime::Alert_Effect::Alert_Effect_DETOUR);
```

Comment:
It is a `Effect DETOUR`
https://github.com/CanalTP/navitia/pull/2689/files#diff-5675d2990ed70bf3b28b8586e5e566eaR2377

Reply:
It's a leftover from an old way of choosing effect (still in `get_calculated_trip_effect` which is deprecated), it has changed in Kirin since then.
It only affects this test, so it's fine to let it now. I will just change it in my PR that focuses on cleaning tests.